problem3.swift

```diff
@@ -1,3 +1,20 @@
 // See problem3.README
+import Foundation
 
 let n = Int(readLine()!)!
+var i = 2
+var isPrime = true
+// Check the remainder of n divided by all integers less than sqrt(n) and break if any divide evenly
+testPrime: while i < Int(sqrt(Double(n))) {
```

Comment:
You don't actually need this label to break out of this loop since it is the inner most loop around the if statement with the break statement. Just writing `break` will break out of the inner most loop. However, this is good to know for future, bigger programs.